packages/cw721/src/msg.rs

```diff
@@ -140,21 +192,60 @@ pub enum Cw721QueryMsg<TMetadataExtension> {
     #[returns(NumTokensResponse)]
     NumTokens {},
 
-    #[returns(CollectionInfo)]
+    #[deprecated(
+        since = "0.19.0",
+        note = "Please use GetCollectionInfoAndExtension instead"
+    )]
+    #[returns(CollectionInfoAndExtensionResponse<TCollectionExtension>)]
+    /// Deprecated: use GetCollectionInfoAndExtension instead! Will be removed in next release!
     ContractInfo {},
 
+    /// Returns `CollectionInfoAndExtensionResponse`
+    #[returns(CollectionInfoAndExtensionResponse<TCollectionExtension>)]
+    GetCollectionInfoAndExtension {},
+
+    /// returns `AllInfoResponse` which contains contract, collection and nft details
+    #[returns(AllInfoResponse)]
+    GetAllInfo {},
+
+    /// Returns `CollectionExtensionAttributes`
+    #[returns(CollectionExtensionAttributes)]
+    GetCollectionExtensionAttributes {},
+
+    #[deprecated(since = "0.19.0", note = "Please use GetMinterOwnership instead")]
```

Comment:
@jhernandezb all CI tests pass. except linter complaining here:

```sh
error: use of deprecated variant `msg::Cw721QueryMsg::Ownership`: Please use GetMinterOwnership instead
   --> packages/cw721/src/msg.rs:218:5
    |
218 |     Ownership {},
    |     ^^^^^^^^^
```

In other repos it doesn complain:
https://github.com/CosmWasm/cosmwasm/blob/430822653351770a89196be1ee7f832f65f37b16/packages/std/src/serde.rs#L17